gremlinapi/templates.py

```diff
@@ -13,8 +13,14 @@
     HTTPError,
 )
 
```

Comment:
TODO: Evaluate deprecating this interface: IIRC this has now `officially` been deprecated from the API.

Reply:
Will deprecate the entire templates.py file